src/assets/assetMetrics.js

```diff
@@ -356,6 +357,7 @@ class AssetMetrics {
    * @param {string} assetMetricValue.effectiveStartDate ISO 8601 Extended Format date/time string
    * @param {string} [assetMetricValue.notes]
    * @param {string} assetMetricValue.value
+   * @param {boolean} assetMetricValue.is_estimated Whether the value is an estimation or a true reading
```

Comment:
Similar question about camel case. Also should this have brackets around it since it's optional? Since there's a default value in the db?